ci-operator/step-registry/upi/deprovision/powervs/cluster/upi-deprovision-powervs-cluster-commands.sh

```diff
@@ -55,24 +55,6 @@ function ic() {
   HOME=${IBMCLOUD_HOME_FOLDER} ibmcloud "$@"
 }
 
-function setup_jq() {
-  if [ -z "$(command -v jq)" ]
-  then
-    echo "jq is not installed, proceed to installing jq"
-    curl -L "https://github.com/jqlang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64" -o /tmp/jq && chmod +x /tmp/jq
-  fi
-}
-
-function setup_ibmcloud_cli() {
-  if [ -z "$(command -v ibmcloud)" ]
-  then
-    echo "ibmcloud CLI doesn't exist, installing"
-    curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
-  fi
-
-  retry "ic config --check-version=false"
-  retry "ic version"
```

Comment:
We probably want the ic config --check-version=false and ic version just for debug issues.